### PR TITLE
add STDC pretrained model on Cityscapes

### PR DESCRIPTION
**Pull request by michaelowenliu**

### CODE REVIEW

File: configs/stdcseg/stdc2_seg_cityscapes_1024x512_80k.yml [hunk 1]
```diff
@@ -1,60 +1,7 @@
...
-  pretrained: null
-
-train_dataset:
-  type: Cityscapes
-  dataset_root: data/cityscapes
-  transforms:
-    - type: ResizeStepScaling
-      min_scale_factor: 0.125
-      max_scale_factor: 1.5
-      scale_step_size: 0.125
```
Comment (shiyutang) [inline configs/stdcseg/stdc2_seg_cityscapes_1024x512_80k.yml:21]: 原版的数据增强部分和cityscpaps的不同，我们就是按照cityscapes训练的么

File: configs/stdcseg/stdc2_seg_cityscapes_1024x512_80k.yml [hunk 2]
```diff
@@ -1,60 +1,7 @@
...
-    - type: RandomPaddingCrop
-      crop_size: [1024, 512]
-    - type: RandomHorizontalFlip
-    - type: RandomDistort
-      brightness_range: 0.5
-      contrast_range: 0.5
-      saturation_range: 0.5
```
Comment (shiyutang) [inline configs/stdcseg/stdc2_seg_cityscapes_1024x512_80k.yml:28]: 这部分cityscapes都是0.4

File: configs/stdcseg/stdc2_seg_cityscapes_1024x512_80k.yml [hunk 3]
```diff
@@ -1,60 +1,7 @@
...
-    - type: Normalize
-      mean: [0.485, 0.456, 0.406]
-      std: [0.229, 0.224, 0.225]
-  mode: train
```
Comment (shiyutang) [inline configs/stdcseg/stdc2_seg_cityscapes_1024x512_80k.yml:32]: normalize的参数也不同

File: configs/stdcseg/stdc2_seg_cityscapes_1024x512_80k.yml [hunk 4]
```diff
@@ -1,60 +1,7 @@
...
-
-val_dataset:
-  type: Cityscapes
-  dataset_root: data/cityscapes
-  transforms:
-    - type: Normalize
-      mean: [0.485, 0.456, 0.406]
-      std: [0.229, 0.224, 0.225]
-  mode: val
```
Comment (shiyutang) [inline configs/stdcseg/stdc2_seg_cityscapes_1024x512_80k.yml:41]: 同normalize 参数

**Review by shiyutang — APPROVED** [review]
